app/src/main/java/org/kinecosystem/kinit/view/earn/QuestionDualImageFragment.kt

```diff
@@ -26,7 +30,11 @@ class QuestionDualImageFragment : BaseFragment() {
             onInvalidData()
         }
         val binding = DataBindingUtil.inflate<QuestionDualImageFragmentBinding>(inflater, R.layout.question_dual_image_fragment, container, false)
-        binding.model = QuestionDualImageViewModel(questionIndex, activity as QuestionnaireActions?)
+        binding.model = QuestionDualImageViewModel(questionIndex, activity as QuestionnaireActions?, object : OnUpdateObservable {
+            override fun onValueChanged() {
+                binding.executePendingBindings()
```

Comment:
Worth adding a comment explaining why we call `execturePendingBundings() `for future generations.